src/Mod/TechDraw/Gui/DlgPrefsTechDraw2.ui

```diff
@@ -504,6 +504,26 @@
           </property>
          </widget>
         </item>
+        <item row="0" column="0">
+         <widget class="QLabel" name="label_13">
+          <property name="text">
+           <string>Line Group </string>
```

Comment:
@wandererfan there is a trailing whitespace in this string. This can create confusion for the translators. Is it possible to omit or is it a cosmetic workaround?